gems/aws-eventstream/lib/aws-eventstream/decoder.rb

```diff
@@ -63,26 +63,31 @@ class Decoder
       include Enumerable
 
       ONE_MEGABYTE = 1024 * 1024
+      private_constant :ONE_MEGABYTE
 
       # bytes of prelude part, including 4 bytes of
       # total message length, headers length and crc checksum of prelude
       PRELUDE_LENGTH = 12
+      private_constant :PRELUDE_LENGTH
```

Comment:
I like the private_constant....we should have done these before.  However, since this was previously exposed to customers and this class isn't listed as api private we should leave these constants exposed unfortunately.